### ISSUE TITLE
Feature request: Sync Screenshot plugin taken pictures to any cloud image service.

### ISSUE THREAD

**Issue by gccltl** [runelite/runelite#3147]
**Sync Screenshot plugin taken pictures to any cloud image service.**
RuneLite already has Google integration, why not use syncing to Google Drive? Dropbox? Or any other cloud provider.

**Comment by j03l** [comment]
Even having the ability to sync the photos folder would be awesome.

**Comment by deathbeam** [comment]
Dont think this is doable easily. Just symlink your screenshots folder to your Dropbox folder?

**Comment by gccltl** [comment]
@deathbeam Yeah I can do so, but I do not want to have additional programs cluttering over my computer, especially Dropbox. Such a feature could work without any additional programs/tools.

**Comment by deathbeam** [comment]
There is already upload to imgur. If you need to use cloud storage for files, use services made for that, when it is possible with them.

**Comment by gccltl** [comment]
Yes, but you don't get a ability to track the links. It would be nice if they atleast could be written to some log file or sth.
